sdk/servicebus/azure-servicebus/azure/servicebus/common/mixins.py

```diff
@@ -203,6 +209,9 @@ def create_subscription(
         :type max_delivery_count: int
         :param enable_batched_operations:
         :type: enable_batched_operations: bool
+        :param fail_on_exist: Whether to throw an exception if there is already a subscription with same name
```

Comment:
Grammer nit: "... exception if a subscription with the same name already exists"

Reply:
Thanks @KieranBrantnerMagee .

Any ideas about the CI failure https://dev.azure.com/azure-sdk/public/_build/results?buildId=227071&view=logs&jobId=24e84196-b4b8-5123-c7ea-5e853df7a17a&j=24e84196-b4b8-5123-c7ea-5e853df7a17a&t=9348bb1d-06b8-5827-b532-d0a0d6f8589e ?

Reply:
See comment below; am rerunning and will dig deeper if it still fails.  (I'm chuckling at that it's only the _final commit making tiny touchups_ that it decided to block us on, but such is life and software :) )

Reply:
Haha yeah totally agreed.

Is it possible to run the tests locally so I can also try to debug?

Reply:
Good news; as you may have noticed the checks have succeeded below. (Thanks Scott!)

However, our hosted livetest pipelines are still pending a fix, so I haven't been able to run the full test suite on an actual SB yet.  From the above can I infer you've not run them locally (via tox/pytest; for posterity [have a doc breadcrumb](https://github.com/Azure/azure-sdk-for-python/blob/master/CONTRIBUTING.md), apologies for not noticing this sooner ).  If not I can quickly pull down this branch and do so to save you more trouble, no worries regardless.

Thanks for all your patience in this.

Reply:
Hi that's great to see the checks passed.

I tried to run tox locally but failed with some error messages - might be something wrong with WSL? not so sure, I might need to fix this later. If you could quickly run this for me that will be nice. I
```
ERROR: could not install deps [-rdev_requirements.txt, pytest-custom_exit_code, pytest-cov, pytest-xdist, pytest-asyncio; python_version >= '3.5', coverage==4.5.4, ../../../tools/azure-devtools, ../../../tools/azure-sdk-tools]; v = InvocationError('/mnt/e/projects/azure-sdk-for-python/sdk/servicebus/azure-servicebus/.tox/whl/bin/python -m pip install -rdev_requirements.txt pytest-custom_exit_code pytest-cov pytest-xdist \'pytest-asyncio; python_version >= \'"\'"\'3.5\'"\'"\'\' coverage==4.5.4 ../../../tools/azure-devtools ../../../tools/azure-sdk-tools', 1)
```

Reply:
Following up on this:  I pulled your branch and ran the tests locally, everything looks good, there was a tiny fix due to an eccentricity in how standard_subscription works, but the logic of your test was sound.  I'll be pushing that momentarily and approving the PR :)

Reply:
Thanks @KieranBrantnerMagee for doing this